prisma-fmt/src/text_document_completion.rs

```diff
@@ -117,11 +117,11 @@ fn push_ast_completions(
             ast::ModelPosition::Field(_, ast::FieldPosition::Attribute("relation", _, Some(attr_name))),
         ) if attr_name == "onDelete" || attr_name == "onUpdate" => {
             for referential_action in connector.referential_actions(&referential_integrity).iter() {
-                items.push(CompletionItem {
+                completion_list.items.push(CompletionItem {
                     label: referential_action.as_str().to_owned(),
                     kind: Some(CompletionItemKind::ENUM),
-                    detail: None, // what is the difference between detail and documentation?
-                    documentation: Some(Documentation::String(referential_action.documentation().to_owned())),
+                    // what is the difference between detail and documentation?
```

Comment:
what is the difference between detail and documentation? (I'm curious and maybe remove the comment)

Reply:
I still don't know.

Reply:
Docs say
```
detail?: string
A human-readable string with additional information about this item, like type or symbol information.

documentation?: string | MarkdownString
A human-readable string that represents a doc-comment.
```

Reply:
and we currently do not use `detail` in the TS code